src/modules/map/utils.ts

```diff
@@ -53,7 +53,10 @@ function fromSpecialTile(specialTile: SpecialTile): Tile {
 }
 export function addSpecialTiles(tiles: Record<string, Tile>) {
   for (const specialTile of Object.values(specialTiles)) {
-    tiles[specialTile.id] = fromSpecialTile(specialTile)
+    tiles[specialTile.id] = {
+      ...tiles[specialTile.id],
+      ...fromSpecialTile(specialTile)
```

Comment:
This indeed fixes the issue, but I think it still has some unintended consequences like overriding the data from the graph (`tiles`) with the fallback data (`specialTiles`). The purpose of `specialTiles` is to be a fallback for tiles that don't exist on TheGraph. That doesn't happen on Mainnet because every single parcel exist as a token, but on Ropsten most of the roads/plazas/district don't even exist in the blockchain, so without the `specialTiles` data those would look empty. 
I think the best approach to solve the issue is to only use the `specialTiles` data if there's no data for that tile from TheGraph, something like this:

```ts
tiles[specialTile.id] = specialTile.id in tiles
        ? tiles[specialTile.id]
        : fromSpecialTile(specialTile)
```

That would even be more performant since we would not do any merge and wouldn't even compute the `fromSpecialTile` helper when a tile already exists.

What do you think?

Reply:
I completely agree